app/debugbar.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package app
```

Comment:
General comment: I'd prefer to put this under `platform` according to the 5.1 proposal of monorepo: https://docs.google.com/document/d/1dbofEY-JJ65Hh_g93OJB0zIG-ao0fzjGuiwij8V-Tus/edit#heading=h.k00l6h41gnsl

Reply:
Good point, I'll move it to platform.

Reply:
done.